lib/responder.ts

```diff
@@ -52,14 +54,14 @@ export class Responder {
   //todo: overload this function
   sendAck(timeoutMs: any, responsesRemaining?: any, cb?: any): void {
     if (!cb) {
-      cb = _.last(arguments);
-      if (!_.isFunction(cb)) cb = null;
+      cb = last(arguments);
+      if (!isFunction(cb)) cb = null;
       if (timeoutMs === cb) timeoutMs = null;
       if (responsesRemaining === cb) responsesRemaining = undefined;
     }
 
     this.ack.timeoutMs = (timeoutMs > -1) ? timeoutMs : this.ack.timeoutMs;
-    if (_.isUndefined(responsesRemaining)) {
+    if (isUndefined(responsesRemaining)) {
```

Comment:
I agree there is no need a dependency that only do: 
```js

function isUndefined(value) {
  return value === undefined;
}
```
https://github.com/lodash/lodash/blob/4.0.0-npm-packages/lodash.isundefined/index.js#L27-L29